src/panel/poi/blocks/Recycling.jsx

```diff
@@ -0,0 +1,57 @@
+/* globals _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Text, Meter } from 'src/components/ui';
+
+const GREEN = '#87c966';
+const YELLOW = '#ffda12';
+const RED = '#ff3b4a';
+
+const containerTypes = type => {
+  return {
+    glass: _('Glass'),
+    paper: _('Paper, carton'),
+  }[type] || _('Unknown');
+};
+
+const Container = ({ type, filling_level, updated_at }) =>
+  <div className="recycling-container">
+    <Text level="caption" icon="icon_clock" className="u-mb-8">
+      {_(
+        'Updated {datetime}',
+        'recycling', {
+          datetime: Intl.DateTimeFormat(window.getLang(), {
+            year: 'numeric', month: 'numeric', day: 'numeric',
+            hour: 'numeric', minute: 'numeric', second: 'numeric',
+            hour12: false,
```

Comment:
Do we really want to force a 24-hour time for countries where a 12-hour time is used ?

Reply:
Thank you, I removed it. I also fixed a locale problem and removed the display of seconds (I don't think they are critical here :P)